components/riemann/scripts/creation_validation.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
They don't validate creation, but they run as part of the `cloudify.interfaces.validation.creation` interface operation. So we need to make sure it is clear that these scripts will also be run when creating, not just on upgrade (ie. so that we don't forget the `if is_upgrade:` part).

I'm all for changing the names, if you think that is fine?